webApp/src/main/java/com/infoshare/controller/VolunteerController.java

```diff
@@ -55,6 +59,12 @@ public VolunteerController(VolunteerService volunteerService, VolunteerFiltering
     @PostMapping("/filtering")
     public String filtering(@ModelAttribute("volunteerFilterForm") VolunteerFilterForm form,
                             RedirectAttributes redirectAttributes) {
+
+        final String uri = "http://localhost:8081/volunteerSearchStats";
+        RestTemplate restTemplate = new RestTemplate();
```

Comment:
Ten rodzaj kodu jest swietnym kandydatem do uzycia aspektow w Springu: https://nullpointerexception.pl/jak-korzystac-ze-spring-aop/ . Jesli nie chce Ci sie przerobic tego na askpety, to wydziel ten kod do oddzielnego serwisu  i zrob stad jedynie wywolanie tej metody.